src/BinaryRewriting/BclRewriter/project.json

```diff
@@ -0,0 +1,18 @@
+﻿{
+  "dependencies": {
+    "Microsoft.Cci": "4.0.0-beta-23321",
+    "System.Console": "4.0.0-beta-23321",
+    "System.Xml.ReaderWriter": "4.0.10",
+    "System.Linq": "4.0.0-beta-23321",
+    "System.Reflection.TypeExtensions": "4.0.0-beta-23321",
+    "Microsoft.NETCore.Runtime.ApiSets-x64": "1.0.0-beta-22914",
+    "Microsoft.NETCore.Runtime.CoreCLR.ConsoleHost-x64": "1.0.0-beta-22914",
+    "Microsoft.NETCore.Runtime.CoreCLR-x64": "1.0.0-beta-23127"
```

Comment:
Do you need to reference the Runtime packages directly?

If you do need to reference them don't reference the "-64" specific one. Instead reference "Microsoft.NETCore.Runtime" itself.